### PR TITLE
profiling: add flops/bytes for elwise kernels

### LINKED ISSUE #205: PyOpenCL array profiling: Use Kernel.function_name to pick kernels apart
cc @matthiasdiener

### PR DESCRIPTION
**Pull request by matthiasdiener**
Also splits the elwise kernels by name (fixes #205).

### CODE REVIEW

**Comment by inducer** [comment]
(Back to draft because we should be able to get by without a component in PyOpenCL, https://github.com/inducer/pyopencl/pull/417)

**Comment by matthiasdiener** [comment]
Ready for review. The estimate is now fully contained within mirgecom.

File: mirgecom/profiling.py [hunk 1]
```diff
@@ -63,6 +63,16 @@ class NonLoopyProfilekernel:
     name: str
 
 
+@dataclass(eq=True, frozen=True)
+class NonLoopyProfileEvent:
+    """Hold a non-Loopy profile event that has not been seen by the profiler yet."""
```
Comment (inducer) [inline mirgecom/profiling.py:68]:
```suggestion
class NonLoopyProfileEvent:
    r"""Hold a non-Loopy profile event that has not been seen by the profiler yet.
    
    These events typically arise from operations performed on :class:`pyopencl.array.Array`\ s.
    """
```

**Review by inducer — APPROVED** [review]
Thx!

**Pull request merged**